### PR TITLE
Edited and structurized documentation for tensor module

### PR DESCRIPTION
**Pull request by drybalka**
#### Release Notes

NO ENTRY

### CODE REVIEW

File: sympy/tensor/tensor.py [hunk 1]
```diff
@@ -1297,8 +1284,8 @@ class TensorSymmetry(Basic):
     =====
 
     A tensor can have an arbitrary monoterm symmetry provided by its BSGS.
-    Multiterm symmetries, like the cyclic symmetry of the Riemann tensor,
-    are not covered.
+    Multiterm symmetries, like the cyclic symmetry of the Riemann tensor
+    (Bianchi identity), are not covere
```
Comment (Upabjojr) [inline sympy/tensor/tensor.py]: typo?

File: sympy/tensor/tensor.py [hunk 2]
```diff
@@ -1539,7 +1526,7 @@ def tensorhead(name, typ, sym=None, comm=0):
     >>> A(a, -b)
     A(a, -b)
 
-    If no symmetry parameter is provided, assume there are not index
+    If no symmetry parameter is provided, assume there are no index
```
Comment (Upabjojr) [inline sympy/tensor/tensor.py]: indices instead of index?

**Comment by Upabjojr** [comment]
This PR looks good.

File: sympy/tensor/tensor.py (hunk 1 above)
Comment (drybalka) [reply sympy/tensor/tensor.py]: Yes, thanks for noticing!

File: sympy/tensor/tensor.py (hunk 2 above)
Comment (drybalka) [reply sympy/tensor/tensor.py]: Yes, you are right. I am new to this process, so is it better to make the corrections and make a new commit to this pull request or force-push the changes to the existing commit for cleaner history?

File: sympy/tensor/tensor.py [hunk 3]
```diff
@@ -299,7 +294,7 @@ def indices_canon_args(self):
         """
         Returns ``(g, dummies, msym, v)``, the entries of ``canonicalize``
 
-        see ``canonicalize`` in ``tensor_can.py``
+        see ``canonicalize`` in ``tensor_can.py`` in combinatorics module
```
Comment (Upabjojr) [inline sympy/tensor/tensor.py]: I think we have some way to cross-link the documentation. I don't remember exactly how.

**Comment by Upabjojr** [comment]
@drybalka I have merged. Thanks for your contribution. If you wish to go on just keep pushing so that we can open a new PR.

**Pull request merged**